### ISSUE TITLE
Warn users about extension manager hitting external servers

### ISSUE THREAD

**Issue by saulshanabrook** [jupyterlab/jupyterlab#8079]
As mentioned [by @vidartf](https://github.com/jupyterlab/jupyterlab/issues/8043#issuecomment-599611692) it would be good to tell users we are sending requests to an external URL when they enable the extension manager. If we also query github for icons (https://github.com/jupyterlab/jupyterlab/pull/8078) we should mention that as well.

**Comment by echarles** [comment]
Are you seeing an update of the warning text we work on for the black and white listings as described [here](https://github.com/jupyterlab/jupyterlab/issues/7967#issuecomment-599072699)?

**Comment by saulshanabrook** [comment]
Yeah I think we should add something to the copy about making requests to NPM and GitHub.

**Comment by echarles** [comment]
Adding @tgeorgeux to the discussion so he can make a proposal for an improved warning message.
